src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -173,6 +175,16 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
   }
 
   ngOnInit() {
+    // ngx-datatable triggers calculations each time mouse enters a row,
+    // this will prevent that.
+    window.addEventListener(
+      'mouseenter',
+      function(event) {
+        event.stopPropagation();
+      },
+      true
```

Comment:
Hi, because I wasn't sure if it's easy or pretty hard to write a test for it, I just tried and it's not that hard ;)

Feel free to use mine or write a different implementation.

```diff
diff --git a/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts b/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts
index 43e1bc25f7..19c59e5e22 100644
--- a/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts
+++ b/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts
@@ -79,6 +79,13 @@ describe('TableComponent', () => {
     expect(component.userConfig.limit).toBe(1);
   });
 
+  it('should fix', (done) => {
+    fixture.detectChanges();
+    const mouseEvent = new MouseEvent('mouseenter');
+    mouseEvent.stopPropagation = () => done();
+    fixture.debugElement.nativeElement.dispatchEvent(mouseEvent);
+  });
+
   describe('test search', () => {
     const expectSearch = (keyword: string, expectedResult: object[]) => {
       component.search = keyword;
```

Reply:
Done. Thanks.